tests/components/husqvarna_automower/test_number.py

```diff
@@ -0,0 +1,78 @@
+"""Tests for number platform."""
+
+from datetime import timedelta
+from unittest.mock import AsyncMock
+
+from aioautomower.exceptions import ApiException
+from freezegun.api import FrozenDateTimeFactory
+import pytest
+
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import HomeAssistantError
+from homeassistant.helpers import entity_registry as er
+
+from . import setup_integration
+
+from tests.common import MockConfigEntry, async_fire_time_changed
+
+
+async def test_number_states(
+    hass: HomeAssistant,
+    mock_automower_client: AsyncMock,
+    mock_config_entry: MockConfigEntry,
+    freezer: FrozenDateTimeFactory,
+    entity_registry: er.EntityRegistry,
+) -> None:
+    """Test number state."""
+    entity_id = "number.test_mower_1_cutting_height"
+    await setup_integration(hass, mock_config_entry)
+    state = hass.states.get(entity_id)
+    assert state is None  ## disabled by default
+    entity_registry.async_update_entity(entity_id=entity_id, disabled_by=None)
```

Comment:
Use a fixture instead. I don't have it on me, but if you go the the test_sensor.py of Withings, it's at the first test `@pytest.....`